src/app/components/core/notifications/notification-service-modal/notification-service-modal.component.ts

```diff
@@ -430,6 +430,7 @@ export class NotificationServiceModalComponent implements OnChanges {
   }
 
   goToLink() {
-    this.docService.goToNotificationDocLink();
+    const urlSlug = 'configuring-the-notification-service';
```

Comment:
probably @monika-Dianomic shall remove *ngIf="!isNotificationServiceAvailable"  check for this; (i.e. edit mode) for service modal in FOGL-5096